packages/ui-components/src/components/text-field/text-field.base.scss

```diff
@@ -147,6 +148,26 @@ input {
 		height: 100%;
 		user-select: none;
 		padding-left: $spacing-3x;
+		padding-top: 2px;
+		color: var(--text-color-input-default);
+
+		&.invalid {
+			color: var(--border-color-error);
```

Comment:
Should it be the border-color or the text-color?

Reply:
It should be the text color 😃  will update